.github/workflows/ci-pipeline.yml

```
@@ -0,0 +1,29 @@
+name: Continuous Integration
+on:
+  schedule:
+    - cron: '35 0 * * *'
+  push:
+    paths-ignore:
+        - 'README.md'
+  pull_request:
+    paths-ignore:
+        - 'README.md'
+
+jobs:
+  build:
+    name: Build and test
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - run: |
+        docker build --target test --tag my-test-image .
+        docker run --env-file .env.test  my-test-image
```

I'd personally recommend splitting these into two steps:
```suggestion
    - run: docker build --target test --tag my-test-image .
    - run: docker run --env-file .env.test  my-test-image
```

That way, when one of them fails, you'll be able to tell at a glance which one!
(For example, as a dev I can easily imagine that I get used to this step failing because I've broken a test, but then the one time it breaks because my docker image has broken I'll spend half an hour trying to work out why my local tests are still all passing!)